extras/kinesis-asl/src/test/scala/org/apache/spark/streaming/kinesis/KinesisStreamSuite.scala

```diff
@@ -56,6 +56,8 @@ class KinesisStreamSuite extends KinesisFunSuite
   private var ssc: StreamingContext = null
   private var sc: SparkContext = null
 
+  protected val aggregateTestData: Boolean
```

Comment:
Isn it easier to have a parameter in the constructor? Less code while subclassing `class WithAggregationKinesisStreamSuite extends KinesisStreamTests(true)`